frb_codegen/src/generator/dart/mod.rs

```diff
@@ -424,6 +429,12 @@ fn generate_api_func(func: &IrFunc, ir_file: &IrFile) -> GeneratedApiFunc {
         format!("_wire2api_{}", func.output.safe_ident())
     };
 
+    let parse_error_data = if let Some(error_output) = &func.error_output {
+        format!("parseErrorData: _wire2api_{},", error_output.safe_ident())
+    } else {
+        "parseErrorData: null,".to_string()
+    };
```

Comment:
nit: extract "parseErrorData: {}," into the template, just like what we do with parseSuccessData